### ISSUE TITLE
iOS and Android support?

### ISSUE THREAD

**Issue by albo1337** [atiruz/pdfviewer#2]
Hey,

does this support android and ios? Meaning can I use your lib in android and ios to show pdfs?

Best regards
Albo

**Comment by atiruz** [comment]
Yep, you can check the [example](https://github.com/atiruz/pdfviewer/blob/master/example/lib/main.dart)

**Comment by albo1337** [comment]
Could you tell me how to import this in pubspec.yaml?

**Comment by albo1337** [comment]
I implemented it like this in my pubspec.yaml:
```
dependencies:
  flutter:
    sdk: flutter
  pdfviewer:
```
Now I receive this error:
```
Because this_example_app depends on pdfviewer any which requires SDK version >=1.19.0 <2.0.0, version solving failed.
pub get failed (1)
```

I think she has the same error: https://github.com/atiruz/pdfviewer/issues/1#issue-357670232

Thanks for your answer @atiruz

**Comment by atiruz** [comment]
https://pub.dartlang.org/packages/pdfviewer#-installing-tab-

**Comment by albo1337** [comment]
Still receiving the error. Could you update your package please? I already added a pull request.

Thx for the answer @atiruz
